using_images/other_images/jenkins.adoc

```diff
@@ -293,6 +293,16 @@ The default maven agent pod uses `docker.io/openshift/jenkins-agent-maven-35-cen
 the CentOS or RHEL version of the Jenkins image. This variable must be set before Jenkins starts
 the first time for it to have an effect.
 
+* `JENKINS_UC_INSECURE`
++
+Determines whether Jenkins plugins downloads are allowed if the Jenkins Update Center repository 
+uses an invalid SSL certificate. This could be the case if a self hosted repository 
+using self-signed certificate with an unknow CA is used or if an enteprise proxy  
+performs man-in-the-middle interceptions. This variable applies to plugins downloads which may 
+occur during Jenkins image build or if an extension of the Jenkins image is built or if you 
```

Comment:
If the upper case “Jenkins” here are necessary?

Reply:
fixed, I set Jenkins with upper everywhere.